tests/lsf/test_lsf_driver.py

```diff
@@ -0,0 +1,140 @@
+import os
+import shutil
+import sys
+from argparse import ArgumentParser
+from textwrap import dedent
+
+import pytest
+
+from ert_shared.cli import ENSEMBLE_EXPERIMENT_MODE
+from ert_shared.cli.main import run_cli
+from ert_shared.main import ert_parser
+
+
+def test_run_mocked_lsf_queue(tmpdir, source_root):
```

Comment:
Seems like we are doing quite some path operations here. Perhaps we could utilise `pathlib` for increased readability?